### ISSUE TITLE
UX improvement: assume SMTP / IMAP servers + users + passwords to be same on setting new email account

### ISSUE THREAD

**Issue by nekromoff** [nextcloud/mail#2732]
### Feature Request

UX improvement: assume SMTP / IMAP servers to be same, if not requested otherwise

#### Summary

IMAP / SMTP servers are often simplified under one domain (URL) and service depends on an accessing port. Same goes for an user account + a password. Simplified settings are OK, but it takes too long to setup for multiple email accounts.

Advanced settings currently require user to enter separate mail servers for SMTP/IMAP as well as separate user + password. These are most of the time the same, so there could be a checkbox to check to allow to enter different details for both connections (IMAP / SMTP) instead of that being a default screen.

This would reduce number of text inputs present at first and only further advanced config would display more text inputs.

**Comment by ChristophWurst** [comment]
Good point. What we already do is copy username and password from the automatic mode into the respective IMAP and SMTP fields when the user switches to manual mode.

Assuming that most users will fill in the form top to bottom, we could automatically copy the values of server, username and password from the IMAP section to SMTP when these fields are empty otherwise.

**Comment by ChristophWurst** [comment]
Everything relevant can be found in this single component: https://github.com/nextcloud/mail/blob/master/src/components/AccountForm.vue :)
